<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Woordstudie</class>
 <widget class="QWidget" name="Woordstudie">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>850</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>850</width>
    <height>850</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>850</width>
    <height>850</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Woordstudie</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-image: url(&quot;C:/Users/harri/Desktop/Nederlands app/images/arkaplan_resmi.webp&quot;);
    background-repeat: no-repeat;
    background-position: center;
    background-size: cover;
}
/* Başlık Frame'i */
QFrame {
    background: transparent; /* Şeffaf bırak */
    border: none;
}
QTextEdit {
    background-color: #f0f8ff; /* Açık mavi tonunda arka plan */
    border: 2px solid #4682b4; /* Hafif koyu mavi çerçeve */
    border-radius: 8px; /* Köşeleri yuvarlak yap */
    padding: 6px; /* İçeride boşluk bırak */
    font-size: 14px;
    color: #333; /* Yazı rengi */
}


QLabel {
	background: transparent; 
    font-size: 18px;
    font-weight: bold;
    color: #31857F;
}

/* Kullanıcı Girişi Frame'i */
QFrame#inputFrame {
    background: transparent; 
    border-radius: 15px;
}

QLineEdit {
    background-color: #f0f5f9;
    border: 2px solid #5EAAA8;
    border-radius: 8px;
    padding: 6px 10px;
    font-size: 16px;
    color: #333;
}

/* Butonların Bulunduğu Frame */
QFrame#progressFrame {
    background: transparent;
}
QPushButton {
    background-color: #5EAAA8; /* Normal buton rengi */
    color: #F4D03F; /* Sarımsı yazı rengi */
    border-radius: 10px;
	font-family: &quot;Comic Sans MS&quot;;
    padding: 8px 15px;
    font-size: 20px;
    border: 2px solid #31857F;
}

/* Mouse ile üzerine gelince */
QPushButton:hover {
    background-color: #76C7C0; /* Daha açık yeşil */
    border: 2px solid #5EAAA8;
}

/* Tıklayınca */
QPushButton:pressed {
    background-color: #EAF4F4; /* Açık renk */
    border: 2px solid #5EAAA8;
}

/* Butonun şeffaf olmasını önlemek için */
QPushButton:!disabled {
    background: #5EAAA8; 
}
</string>
  </property>
  <widget class="QComboBox" name="comboBox_verwanteVormen">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>300</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_woordselecteer">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>233</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_woordtype">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>163</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="currentText">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>52</x>
     <y>277</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Verwante vormen</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>136</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Kies het woordtype</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>211</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Selecteer een woord</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_metinAra">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>290</y>
     <width>431</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>728</x>
     <y>290</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Zoek</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_duzenle">
   <property name="geometry">
    <rect>
     <x>668</x>
     <y>356</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Wijzigen</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_kaydet">
   <property name="geometry">
    <rect>
     <x>279</x>
     <y>356</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Opslaan</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_sil">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>356</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Verwijderen</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>130</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Beschrijving</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_aciklama">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>167</y>
     <width>531</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#textEdit_aciklama {
    background-color: #f0f5f9; /* Açık sarı tonu (tablodaki gibi) */
    border: 2px solid #5EAAA8; /* Çerçeve */
    border-radius: 8px;
    padding: 6px;
    font-size: 14px;
    color: #333;
}
</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Nederlands">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>487</y>
     <width>781</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Voer een zin in het Nederlands in</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Turkce">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>420</y>
     <width>781</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">    background-color: #F0F8FF;  /* Açık mavi tonları */
    border: 2px solid #5EAAA8;
    border-radius: 8px;
    padding: 6px 10px;
    font-size: 16px;
    color: #333;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Voer een zin in het Turks in</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget_zinnen">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>550</y>
     <width>781</width>
     <height>291</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(246, 255, 205);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>60</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-family: &quot;Comic Sans MS&quot;; /* Daha eğlenceli ve oyunumsu bir font */
    font-size: 25pt; /* Boyutu biraz daha artırdık, daha dikkat çekici olsun */
    font-weight: bold; /* Kalın yazı (eğlenceli duruşu koruyor) */
    color: #F57C00; /* Daha koyu turuncu ton (orijinal #FF9800’dan daha derin) */
}</string>
   </property>
   <property name="text">
    <string>Woordstudie</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Terug">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>10</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #5EAAA8; /* Normal buton rengi */
    color: #F4D03F; /* Sarımsı yazı rengi */
    border-radius: 10px;
	font-family: &quot;Comic Sans MS&quot;;
    padding: 8px 15px;
    font-size: 15px;
    border: 2px solid #31857F;
}

/* Mouse ile üzerine gelince */
QPushButton:hover {
    background-color: #76C7C0; /* Daha açık yeşil */
    border: 2px solid #5EAAA8;
}

/* Tıklayınca */
QPushButton:pressed {
    background-color: #EAF4F4; /* Açık renk */
    border: 2px solid #5EAAA8;
}

/* Butonun şeffaf olmasını önlemek için */
QPushButton:!disabled {
    background: #5EAAA8; 
}
</string>
   </property>
   <property name="text">
    <string>Terug naar startpagina</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
